packages/ng/skeleton/skeleton-index-table/skeleton-index-table.component.ts

```diff
@@ -0,0 +1,21 @@
+import { booleanAttribute, ChangeDetectionStrategy, Component, computed, input, Input } from '@angular/core';
+
+@Component({
+	selector: 'lu-skeleton-index-table',
+	standalone: true,
+	templateUrl: './skeleton-index-table.component.html',
+	styleUrl: './skeleton-index-table.component.scss',
```

Comment:
This file does not contain any declarations, from what I understand.

Should we just not have it?